lib/matplotlib/mpl-data/stylelib/dark_background.mplstyle

```diff
@@ -21,3 +21,13 @@ figure.edgecolor: black
 savefig.facecolor: black
 savefig.edgecolor: black
 
+### Boxplots
+boxplot.boxprops.color: w
```

Comment:
Please use `white`. While technically `w` is allowed, we try to keep style files more readable.